starknet_py/net/account/account.py

```diff
@@ -359,6 +360,60 @@ async def verify_message(self, typed_data: TypedData, signature: List[int]) -> b
         message_hash = typed_data_dataclass.message_hash(account_address=self.address)
         return await self._verify_message_hash(message_hash, signature)
 
+    @staticmethod
+    async def deploy_account(
+        *,
+        address: int,
+        class_hash: int,
```

Comment:
Should we allow for a string class_hash? My point is that in Starkscan or Voyager `class_hashes` are displayed as hex values. New users will take them and be confused why we require int here

Reply:
Python converts hex literals to integers